api/docker-compose.test.yml

```diff
@@ -0,0 +1,14 @@
+version: '3.7'
+
+services:
+  db:
+    image: postgres:12.0-alpine
+    environment:
+      - POSTGRES_DB=prontolista
+      - POSTGRES_USER=prontolista
```

Comment:
Do we have to setup the database for testing as well? You can explain to me later :D